tools/topology/m4/muxdemux.m4

```diff
@@ -36,6 +36,15 @@ define(`N_MUXDEMUX', `MUXDEMUX'PIPELINE_ID`.'$1)
 
 dnl W_MUXDEMUX(name, mux/demux, format, periods_sink, periods_source, core, kcontrol_list)
 define(`W_MUXDEMUX',
+`SectionVendorTuples."'N_MUXDEMUX($1)`_tuples_uuid" {'
+`	tokens "sof_comp_tokens"'
+`	tuples."uuid" {'
+`		SOF_TKN_COMP_UUID'		"c4:b2:68:68:14:30:47:0e:a0:89:15:d1:c7:7f:85:1a"
+`	}'
```

Comment:
It's better here to have a uuid.m4 file that contains all UUIDs and is included by all topologies.
```
# uuid.m4 - set default UUIDs if NOT already set in topology.
# allows users to override the UUID if needed.
ifndef UUID_SRC
define UUID_SRC "blah blah blah"
endif

# and so on
```

Reply:
As discussed, will implement a m4 helper to generate this UUID byte array from the string copied from the firmware source directly.